lib/rubocop/cop/rspec/empty_line_after_subject.rb

```diff
@@ -22,13 +22,8 @@ class EmptyLineAfterSubject < Cop
 
```

Comment:
The message does not make a difference between `subject` and `subject!`, while other cops indicate what hook/example/example group method was used.

Reply:
Good point. I’m adding a commit to fix this oversight.